samples/tables.js

```diff
@@ -566,7 +518,7 @@ function extractTableToGCS(
   filename,
   projectId
 ) {
-  // [START bigquery_extract_gcs]
+  // [START bigquery_extract_table]
```

Comment:
I assume you'll be updating https://cloud.google.com/bigquery/docs/exporting-data to use the new region tag?

Reply:
Yes